src/modules/auth/auth.service.ts

```diff
@@ -31,6 +31,14 @@ class AuthService {
 	public async createUser(
 		user: UserAuthSignUpRequestDto,
 	): Promise<UserAuthResponseDto> {
+		const existingUser = await this.authRepository.findByEmail(user.email);
+		if (existingUser) {
+			throw new HTTPError({
+				message: "Email is already registered",
```

Comment:
```suggestion
				message: "Email is already taken",
```

and move it to exception message enum, do not use literals directly, see other errors